connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -640,11 +645,7 @@ protected static void waitUntilMirrorMakerIsRunning(EmbeddedConnectCluster conne
      * wait for the topic created on the cluster
      */
     protected static void waitForTopicCreated(EmbeddedConnectCluster cluster, String topicName) throws InterruptedException {
-        try (final Admin adminClient = cluster.kafka().createAdminClient()) {
-            waitForCondition(() -> adminClient.listTopics().names().get().contains(topicName), TOPIC_SYNC_DURATION_MS,
-                "Topic: " + topicName + " didn't get created on cluster: " + cluster.getName()
-            );
-        }
+        waitForTopicCreated(cluster.kafka(), topicName);
```

Comment:
Actually I wonder if we should delete the other `waitForTopicCreated()` method instead as it only has a single caller, and the caller can just use `backup` instead of `backup.kafka()`. WDYT?

Reply:
yeah this will unify the code test utils more.

Reply:
updated the test